packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -533,8 +586,48 @@ class TimeInput extends React.Component {
     if ((event.key === '=' || event.key === '+') && !this.props.atMaxDate) {
       const currentTimeValue = this.formatHour(hour, meridiem).concat(':', minute).concat(this.props.showSeconds ? ':'.concat(second) : '');
       if (TimeUtil.validateTime(currentTimeValue, this.props.showSeconds)) {
-        if (minute === '59') {
-          minute = '00';
+        if (inputType === 3 && this.props.showSeconds) {
+          if (second === '59') {
+            second = '00';
+            if (minute === '59') {
```

Comment:
can we convert these hard coded values to constant ?

Reply:
Updated - https://github.com/cerner/terra-framework/pull/1966/commits/4caa69c649e337786153c7767f908c3a07d605e7